entrypoint.sh

```diff
@@ -3,7 +3,7 @@
 set -eo pipefail
 
 IFS=
-OUTPUT="$(dnscontrol "$1")"
+OUTPUT="$(dnscontrol "$1" --config "$INPUT_DNSCONTROL_CONFIG_FILE" --creds "$INPUT_DNSCONTROL_CREDS_FILE")"
```

Comment:
@koenrh This format is working for me locally but is failing when I try to test this branch in Actions. Do you see anything I am missing?

![image](https://user-images.githubusercontent.com/20355/97212508-e5a10380-1796-11eb-93ef-940e58d7b32e.png)